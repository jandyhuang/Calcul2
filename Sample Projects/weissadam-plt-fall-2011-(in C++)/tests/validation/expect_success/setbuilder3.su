function B [int b] returns int
{ 

 set A = { -1 ... 1 };
 set B = { (A, b) | b in {-1, 1} };
 set C = { ( { -1, 1 }, -1 ), ( { -1, 1 }, 1 ) };

 if ( B == C ) then
 {
   return 0;
 }
 else
 {
   return 1;
 } 

}
