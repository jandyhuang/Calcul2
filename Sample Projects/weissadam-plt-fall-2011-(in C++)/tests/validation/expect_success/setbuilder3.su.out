B set_seq
C set_seq
A set_seq
b int
